Cisco/cisco-ios/ingest/parser.yml

```
@@ -7,8 +7,9 @@ pipeline:
         pattern: "(?:%{HOSTNAME:hostname}: )?%{CISCO_HEADER}: %{DATA:description}"
         custom_patterns:
           CISCO_HEADER: "%{CISCO_DATESTAMP}: %{MSG_CODE}"
-          CISCO_TIMESTAMP: "%{MONTH} %{MONTHDAY}( %{YEAR})? %{TIME}"
-          CISCO_DATESTAMP: "%{CISCO_TIMESTAMP:date}( %{WORD:timezone})?"
+          CISCO_TIMESTAMP_1: "%{MONTH} %{MONTHDAY}( %{YEAR})? %{TIME}"
+          CISCO_TIMESTAMP_2: "%{MONTH}  %{MONTHDAY}( %{YEAR})? %{TIME}"
+          CISCO_DATESTAMP: "(%{CISCO_TIMESTAMP_1:date})?(%{CISCO_TIMESTAMP_2:date})?( %{WORD:timezone})?"
```

I suggest simplifying the change:
```suggestion
          CISCO_TIMESTAMP: "%{MONTH}\\s+%{MONTHDAY}( %{YEAR})? %{TIME}"
          CISCO_DATESTAMP: "%{CISCO_TIMESTAMP:date}( %{WORD:timezone})?"
```